build.gradle

```diff
@@ -70,3 +70,8 @@ project.ext {
 task wrapper(type: Wrapper) {
     gradleVersion = "2.11"
 }
+
+// free build hack: common functions
+def doFreeBuild() {
+    return System.properties["freeBuildHackDoFreeBuild"] == "true"
```

Comment:
I like it, but ``freeBuildHackDoFreeBuild`` is just horrible.
Couldn't we shorten that to just ``freeBuild``, i.e. ``-DfreeBuild=true`` would be passed?

Looks like F-Droid recipes only supports setting gradle/project properties (``-P``) [1].
Maybe ``return hasProperty("freeBuild")``? (we can omit ``=true``)

--
[1] https://f-droid.org/manual/fdroid.html#Build